muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/draggable.tsx

```diff
@@ -271,6 +281,11 @@ export default class Draggable extends React.Component<DraggableProps, Draggable
     }
   }
   onRootSeletEnd(e: MouseEvent | TouchEvent){
```

Comment:
onRootSelectEnd?

Reply:
Haven't changed that line but the element to drag is called the root in that component, the function triggers on mouse event and touch event; so it's a select event, so the function ended up named on root select end, referring to both mouseend and touchend.

Reply:
**onRootSeletEnd** is not the same as suggested **onRootSelectEnd**. Find the typo. Might be better to point the typo in the future if that was the case here.

Reply:
onRootSeletEnd is not the same as suggested onRootSelectEnd. Find the typo. Might be better to point the typo in the future if that was the case here.